dbms/src/test/java/org/polypheny/db/cypher/AggregateTest.java

```diff
@@ -122,15 +127,14 @@ public void singleAvgAggregateTest() {
 
 
     @Test
-    @Ignore // own min max impl necessary
```

Comment:
Can you enable this test?
However, something seems to be wrong with this class anyway, since this @Ignore is Junit 4... This should thrown an error.

Reply:
https://github.com/polypheny/Polypheny-DB/pull/464#discussion_r1520466883

Reply:
This change removes `@Ignore`.